interactive_setup.sh

```diff
@@ -8,6 +8,9 @@ domain=aam-digital.com
 prefix=ndb-
 # Location where Keycloak is running
 source "./keycloak/.env"
+# Data for UptimeRobot
+uptimeRobotAPIKey=
+uptimeRobotAlertID=
```

Comment:
could/should this be sourced from an .env also, to avoid merge conflicts on future updates?

Reply:
Good idea